### PR TITLE
adds Select in course_type_selector

### PR DESCRIPTION
**Pull request by shivanshbindal9**
select

### CODE REVIEW

File: app/assets/javascripts/components/overview/course_type_selector.jsx [hunk 1]
```diff
@@ -38,7 +42,8 @@ const CourseTypeSelector = createReactClass({
       Editathon: 'Edit-a-thon',
       BasicCourse: 'Generic Course',
       ArticleScopedProgram: 'Article Scoped Program',
-      FellowsCohort: 'Wikipedia Fellows Cohort'
+      FellowsCohort: 'Wikipedia Fellows Cohort',
+      TestCase: 'Test Case'
```
Comment (ragesoss) [inline app/assets/javascripts/components/overview/course_type_selector.jsx]: This was just for testing?

**Comment by ragesoss** [comment]
(see also the comment I accidentally left at the issue: #2263)

**Comment by shivanshbindal9** [comment]
I have added tests for Select in course_type_selector !! @ragesoss

**Comment by ragesoss** [comment]
Cool! I think the styling could be improved; the current style we're injecting into the react-select element is meant for multiselect cases where we need to show the division between each selected option, but with a single select, the heavy styling of the selected option is distracting.

**Comment by shivanshbindal9** [comment]
mnpr
Changed the styling !! @ragesoss

**Comment by ragesoss** [comment]
This looks great! If you'd like to move on to some of the other `<select>` cases, please do.

**Comment by ragesoss** [comment]
I'll merge once the build finishes, assuming no problems turn up.

**Pull request merged**